cognitiveservicesPlugin/README.md

```diff
@@ -238,8 +238,8 @@ constructor(private cognitiveServices: CognitiveServices) { }
 // Stop the recognition process (iOS only)
 this.cognitiveServices.StopListening();
 
-// Speak the text in the speakText variable
-this.cognitiveServices.SpeakText(speakText).then(
+// Speak the text in the startSpeaking variable
+this.cognitiveServices.startSpeaking(startSpeaking).then(
```

Comment:
Should the variable be something other than startSpeaking?  It could be confusing.